met-web/src/components/engagement/form/RichTextEditor.tsx

```diff
@@ -60,7 +60,7 @@ const RichTextEditor = ({
 
     return (
         <FormControl fullWidth>
-            <MetPaper style={{ borderColor: `${error ? '#d32f2f' : '#606060'}` }}>
+            <MetPaper style={{ borderColor: `${error ? '#d32f2f' : ''}` }}>
```

Comment:
why? Don't we want this border to be red when there's error?

Reply:
the border is still red when there is an error it is just going to inherit the other border style when there is no error

Reply:
ohhh right! Then let's something like sx={[ error && { borderColor:...}]}. This way borderColor is not set to a bad value like '' when there's no error
![image](https://user-images.githubusercontent.com/91914654/219207244-d5ad1348-1cf1-44a4-a90e-af87db404e0a.png)